circle.yml

```diff
@@ -30,6 +30,11 @@ test:
     - ls -agolf _site/ # Ensure that build:website worked
 
 deployment:
+  add-headers:
```

Comment:
Let's be sure to remove this new env and deployment prior to merging in. Happy to leave in for now, but not something we want to keep for long term.